server/auth/index.js

```diff
@@ -0,0 +1,22 @@
+const passport = require('passport');
+const passportJWT = require('passport-jwt');
+const JwtStrategy = passportJWT.Strategy;
+const ExtractJwt = passportJWT.ExtractJwt;
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
Use preload method for dotenv and remove any usage in code

Reply:
Did you mean on this: `require('dotenv').load();` ??

Reply:
Nevermind, dotenv preloaded